02week/pigLatin.js

```diff
@@ -9,11 +9,22 @@ const rl = readline.createInterface({
 
 
 function pigLatin(word) {
-
-  // Your code here
-
+  // make (word) all lowercase
+  const lowerThatCase = word.toLowerCase().trim();
+  // make that into array
+  const splitWord = lowerThatCase.split('');
+  // check if any vowels if not move on to normalcy
+  if (splitWord[0] === 'a' || splitWord[0] === 'e' || splitWord[0] === 'i' || splitWord[0] === 'o' || splitWord[0] === 'u'){
+    return lowerThatCase + 'yay';
+  } else {
```

Comment:
This else statement runs after the first letter is determined to not be a vowell, but then assumes the next letter is.